client/verta/verta/client.py

```diff
@@ -1040,6 +1044,10 @@ def create_registered_model(self, name=None, desc=None, labels=None, workspace=N
             Visibility to set when creating this registered model. If not
             provided, an appropriate default will be used. This parameter
             should be preferred over `public_within_org`.
+        action_type : :mod:`~verta.registry.action_type`, default :class:`~verta.registry.action_type.Unknown`, optional
+            Action type of the registered_model.
+        data_type : :mod:`~verta.registry.data_type`, default :class:`~verta.registry.data_type.Unknown`, optional
+            Data type of the registered_model.
```

Comment:
We don't need to mention `Unknown` since users shouldn't need to be aware of it at all! See my comment on [action_type/_unknown.py](https://github.com/VertaAI/modeldb/pull/3079/files#diff-9ca7c80e294a86b981b41c413b37c392c8f79a538e649d1af57f2cea2ab967b1).

```suggestion
        action_type : :mod:`~verta.registry.action_type`, optional
            Action type of the registered_model.
        data_type : :mod:`~verta.registry.data_type`, optional
            Data type of the registered_model.
```